### DISCUSSION TITLE
how to set FuturesContract maker_fee and taker_fee

### DISCUSSION THREAD

**Discussion by naught2020** [nautechsystems/nautilus_trader#1709] [Q&A] [answered]
Hi! I am trying to load external historical futures contract OHLC data into backtest engine. My question is how to set FuturesContract maker_fee and taker_fee? When I try to set them during creating FuturesContract instrument, I get error"AttributeError: attribute 'maker_fee' of 'nautilus_trader.model.instruments.base.Instrument' objects is not writable", so how to set commission fee for FuturesContract?
btw, I am using nautilus-trader  1.194.0 Beta
thanks a lot for your advice!

**Comment by cjdsellers** [comment, marked as answer]
Hi @naught2020

This is a known issue, currently tracked by [this ticket](https://github.com/orgs/nautechsystems/projects/3?pane=issue&itemId=67641935).
